.travis.yml

```diff
@@ -1,14 +1,58 @@
 language: r
 cache: packages
-r:
-- 3.3
-- oldrel
-- release
-- devel
 after_success:
 - Rscript -e 'covr::codecov()'
 - bash pkgdown.sh
-env:
-  global:
-    - _R_CHECK_LENGTH_1_LOGIC2_=true
-    - secure: BWvMPmOiTzFNfFz3Ok+WHk6FjNNjK8qV5QohBhS0bgsMgeCrx4Zu3IXp42cXaEzzCe/VEnnEyYYTkZROS3mdLRAlOHnEHvCJMtETOODOK6EFGUFDv+C7AXZyqbdoNPrGrW5QKppHAVbTuL/A9z+tnBaeyG68GhOmOFBk4mciHNc=
+
+# matrix to include a windows build this can be removed when 
+# https://github.com/travis-ci/travis-build/pull/1538 is merged in
+# the windows build steps are similar to what's in the PR and the process other
+# oses use
+matrix:
+  include:
+  - os: linux
+    r: 3.3
+    env:
+      global: 
+        - _R_CHECK_LENGTH_1_LOGIC2_=true
+        - secure: BWvMPmOiTzFNfFz3Ok+WHk6FjNNjK8qV5QohBhS0bgsMgeCrx4Zu3IXp42cXaEzzCe/VEnnEyYYTkZROS3mdLRAlOHnEHvCJMtETOODOK6EFGUFDv+C7AXZyqbdoNPrGrW5QKppHAVbTuL/A9z+tnBaeyG68GhOmOFBk4mciHNc=
+  - os: linux
+    r: oldrel
+    env:
+      global: 
+        - _R_CHECK_LENGTH_1_LOGIC2_=true
+        - secure: BWvMPmOiTzFNfFz3Ok+WHk6FjNNjK8qV5QohBhS0bgsMgeCrx4Zu3IXp42cXaEzzCe/VEnnEyYYTkZROS3mdLRAlOHnEHvCJMtETOODOK6EFGUFDv+C7AXZyqbdoNPrGrW5QKppHAVbTuL/A9z+tnBaeyG68GhOmOFBk4mciHNc=
+  - os: linux
+    r: release
+    env:
+      global: 
+        - _R_CHECK_LENGTH_1_LOGIC2_=true
+        - secure: BWvMPmOiTzFNfFz3Ok+WHk6FjNNjK8qV5QohBhS0bgsMgeCrx4Zu3IXp42cXaEzzCe/VEnnEyYYTkZROS3mdLRAlOHnEHvCJMtETOODOK6EFGUFDv+C7AXZyqbdoNPrGrW5QKppHAVbTuL/A9z+tnBaeyG68GhOmOFBk4mciHNc=
```

Comment:
This secure token should only be needed for the `release` version because it's used to build the pkgdown site and only on "release": https://github.com/Crunch-io/rcrunch/blob/master/pkgdown.sh#L5